user/views.py

```diff
@@ -1,3 +1,68 @@
+import datetime
+import json
+import bcrypt
+import jwt
+
 from django.shortcuts import render
+from django.views     import View  
+from django.http      import HttpResponse, JsonResponse
+
+from my_settings      import SECRET_KEY
+from user.models      import User, Gender, SkinInfo
+
+
+class SignUpView(View):
+    def post(self, request):
+        user_data = json.loads(request.body)
+        try:
+            if User.objects.filter(account = user_data['account']).exists():
+                return JsonResponse({'message':'DUPLICATED ID'}, status=400)
```

Comment:
```suggestion
                return JsonResponse({'message':'DUPLICATED_ID'}, status=400)
```
띄어쓰기는 언더바(`_`)로 구분해주세요.